### DISCUSSION TITLE
watermark utf8 support

### DISCUSSION THREAD

**Discussion by suntong** [libvips/libvips#3771] [Ideas]
I use watermark with Chinese/UTF8 (via [imaginary](https://github.com/h2non/imaginary)), and this is what I got :

![image](https://github.com/libvips/libvips/assets/422244/3772ee1d-f282-4c92-af12-32638dcd8592)

This is running from a server that has Chinese fonts (notos) and supports Chinese/UTF8 characters. 

Does the watermark have the utf8 support please?

**Comment by jcupitt** [comment]
Hello @suntong,

Yes, it should work. I would guess you have some kind of fontconfig issue.

On this PC (ubuntu 23.10) I can run:

```
$ vips text x.png 自增主键 --dpi 600
```

To generate:

![x](https://github.com/libvips/libvips/assets/580843/751dc8e7-fd28-4cda-b205-8fbb978e0bf6)

**Reply by suntong** [reply]
Wow, thanks. I'll look into my imaginary fontconfig.

**Reply by jcupitt** [reply]
You could ask on the imaginary tracker, they might be help.
